scripts/yocto-build-and-test.sh

```diff
@@ -124,7 +124,7 @@ FILESEXTRAPATHS_prepend_pn-mender-binary-delta := "$WORKSPACE/mender-binary-delt
 PREFERRED_VERSION_pn-mender-binary-delta = "$mender_binary_delta_version"
 EOF
 
-    local mender_monitor_version=$(tar -xf $WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz ./mender-monitor/.version -O | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
+    local mender_monitor_version=$(tar -Oxf $WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz ./mender-monitor/.version -O | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
```

Comment:
@kacf this is onpurpose, right? as it is now, `cat .version` will never match it, since it is the short git commit hash. this is when we release, right?